mkt/account/serializers.py

```diff
@@ -137,3 +138,12 @@ class UserSerializer(AccountSerializer):
     class Meta:
         model = UserProfile
         fields = ('display_name', 'resource_uri')
+
+
+class GroupsSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(source='pk', required=False, read_only=False)
```

Comment:
I'll see if I can remove it - though its used in a few other serializers (search for `source='pk'`) so I guessed it was needed if you explicitly wanted id to be available.